iis-common/src/main/java/eu/dnetlib/iis/common/string/LenientComparisonStringNormalizer.java

```diff
@@ -68,12 +88,12 @@ public String normalize(String value) {
         }
         
         
-        String result = null;
-        
-        result = removeNonLetterDigitCharacters(value);
+        String result = value;
         
         result = DiacriticsRemover.removeDiacritics(result);
         
+        result = removeNonLetterDigitCharacters(result);
+        
```

Comment:
Yes. There is problem with accented letters that consists of two chars. One for letter and the second one for accent. `removeNonLetterDigitCharacters` will change the second one into space.
So normalizer in old version will translate for example:
`University of São Paulo` to `university of sa o paulo`